pymtl3/passes/PrintWavePass.py

```diff
@@ -0,0 +1,88 @@
+"""
+========================================================================
+PrintWavePass.py
+========================================================================
+#Print the most sigficant number of signals in wave form.
+.state(or State) is processed in state form.
+#top object uses top.printwave(top) to print.
+
+Author : Kaishuo Cheng
+Date   : Oct 4, 2019
+"""
+
+from __future__ import absolute_import, division, print_function
+import time
+from collections import defaultdict
+from copy import deepcopy
+import py
+import sys
+from pymtl3.dsl import Const
+from pymtl3.passes.BasePass import BasePass, PassMetadata
+
+
+class PrintWavePass( BasePass ):
+
+  def __call__( self, top ):
+    setattr(top,"printwave",helpprint)
+
+def helpprint(self):
```

Comment:
The current codebase uses `underscore_separated_lower_case_names` for functions, methods, and variable names. I think new PR's should also adopt this style for consistency.